lib/src/utils/ne_utils.dart

```diff
@@ -0,0 +1,26 @@
+class NepaliLangUtils {
+  static getNepaliNumber(int num) {
+    final List<String> nepaliDigits = [
+      '०',
+      '१',
+      '२',
+      '३',
+      '४',
+      '५',
+      '६',
+      '७',
+      '८',
+      '९'
+    ];
+
+    String result = '';
+    String numberString = num.toString();
+
+    for (int i = 0; i < numberString.length; i++) {
+      int digit = int.parse(numberString[i]);
```

Comment:
It's minor optimization, but things can add up, I can not ensure any specific usage patterns from users so assuming a screen with multiple number inputs, when a language change occurs these calculations will add up.

I think alongside with the `StringBuffer`, we can use `numberString.charCodeAt(i) - '0'.charCodeAt(0)` would be faster to calculate digit index.